### PR TITLE
WIP: docs: adds acc test setup

### PR DESCRIPTION
**Pull request by mavogel**
## If applied, this commit will...
enhance the documentation for an acceptance test setup for gitlab and gitab

## Why is this change being made?
Contributors can test releasing, brews and scoops locally and set-up quickly a local test environment.

### Provide links to any relevant tickets, URLs or other resources
- https://github.com/goreleaser/goreleaser/pull/1547#issuecomment-731347586

### CODE REVIEW

**Review by caarlos0 — COMMENTED** [review]
looking good so far, want to add more things to this PR?

**Comment by mavogel** [comment]
yes, give me 1-2 days. bit busy atm

File: www/docs/customization/scoop.md [hunk 1]
```diff
@@ -14,6 +14,7 @@ scoop:
   # Template for the url which is determined by the given Token (github or gitlab)
   # Default for github is "https://github.com/<repo_owner>/<repo_name>/releases/download/{{ .Tag }}/{{ .ArtifactName }}"
   # Default for gitlab is "https://gitlab.com/<repo_owner>/<repo_name>/uploads/{{ .ArtifactUploadHash }}/{{ .ArtifactName }}"
+  # Default for gitea is "https://gitea.com/<repo_owner>/<repo_name>/releases/download/{{ .Tag }}/{{ .ArtifactName }}"
   # Gitea is not supported yet, but the support coming
```
Comment (6543) [inline www/docs/customization/scoop.md:18]: look's like I missed this one :/ ?
```suggestion
```

**Comment by caarlos0** [comment]
will cherry pick these commits into master just so they go with the next release :)

**Comment by mavogel** [comment]
When I run the script `acc-tests/local-gitea.sh` I get the error at the end for the command `ssh -vvvT git@localhost -p 222`
```sh
kex_exchange_identification: Connection closed by remote host
```
It is also not possible to `git push`

## Current investigation
- seems similar to https://github.com/wkulhanek/docker-openshift-gitea/issues/9
- `gitea --version` is `Gitea version 1.14.0+dev-220-g0de546009`

@6543 do you have any idea if this could a gitea issue? Or is more related to ssh

**Comment by caarlos0** [comment]
It seems like this will be quite a lot of code... do you think its better if we put it in another repo?

**Comment by mavogel** [comment]
Yes, TBH it would make sense to have a separate repo for this. Can you create one under the goreleaser org please?

**Comment by caarlos0** [comment]
> Yes, TBH it would make sense to have a separate repo for this. Can you create one under the goreleaser org please?

sure thing :)

https://github.com/goreleaser/acceptance-tests

not sure about the name, let me know if you have a better one :)

**Comment by mavogel** [comment]
migrated to https://github.com/goreleaser/acceptance-tests/pull/1

**Pull request closed without merging**